frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -18,14 +19,44 @@ export interface OnboardingLogicProps {
 
 export enum OnboardingStepKey {
     PRODUCT_INTRO = 'product_intro',
-    SDKS = 'sdks',
-    BILLING = 'billing',
-    OTHER_PRODUCTS = 'other_products',
+    INSTALL = 'install',
+    PLANS = 'plans',
```

Comment:
Changing these means our insights will change because they filter on URL, might want to make an action to join them together. Or just have a different enum that maps the step key to a proper name.

Reply:
Action for install step: https://us.posthog.com/project/2/data-management/actions/66284

Action for billing step: https://us.posthog.com/project/2/data-management/actions/66285

Reply:
PA Billing: https://us.posthog.com/project/2/data-management/actions/66286
SR Billing: https://us.posthog.com/project/2/data-management/actions/66287
Surveys Biling: https://us.posthog.com/project/2/data-management/actions/66288
FF Billing: https://us.posthog.com/project/2/data-management/actions/66289

Reply:
Created product specific ones for install step as well.